R/ConceptSet.R

```diff
@@ -332,17 +332,11 @@ resolveConceptSetId <-
            formatName = FALSE) {
     .checkBaseUrl(baseUrl)
     
-    readCsv <- function(file) {
-      concepts <- read.csv(
-        file = substring(file, first = 3),
-        header = FALSE,
-        sep = ",",
-        strip.white = TRUE,
-        blank.lines.skip = TRUE,
-        skipNul = TRUE
-      )
-      names(concepts) <- as.character(apply(concepts[1, ], 1, paste))
-      concepts <- concepts[-1, ]
+    .readCsv <- function(file) {
+      concepts <- readr::read_csv(file = file)
+      names(concepts) <-
+        snakecase::to_lower_camel_case(names(concepts))
```

Comment:
I would prefer we standardize everywhere to using SqlRender to deal with case transformations.

Reply:
Thanks @schuemie this will require enhancement to SqlRender where it can convert an output with space such as 'this makes me happy' to 'thisMakesMeHappy'. SqlRender does not have this function currently, it can only convert snakeCase -> camelCase.

Reply:
Well, in this case, why not create a simple custom function in this package, instead of adding the dependency?

Reply:
@schuemie i think this should do it. it worked in my tests
`string %>% 
  stringr::str_squish() %>% 
  stringr::str_to_lower() %>%
  stringr::str_to_title() %>%
  stringr::str_replace_all(pattern = " ", replacement = "")`

I committed the changes here (dependency on snakeCase package is now removed)
https://github.com/OHDSI/ROhdsiWebApi/pull/32/commits/5be55dfb90998141fd7e521ed60882b1a2c66f9a